src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -179,27 +179,24 @@ export class DictionaryModal extends React.Component {
     return (
       <div>
         <Modal
-          show={this.props.show}
+          isOpen={this.props.show}
           onHide={this.props.modalhide}
```

Comment:
@Duncanian In reactstrap `onHide` is `onClosed` and `dialogClassName` is `modalClassName`. For more about modal component props check this https://reactstrap.github.io/components/modals/

Reply:
I have done it.